Library/Homebrew/rubocops/lines_cop.rb

```diff
@@ -67,7 +67,69 @@ def audit_formula(_node, _class_node, _parent_class_node, body_node)
             next unless block_arg.source.size>1
             problem "\"inreplace <filenames> do |s|\" is preferred over \"|#{block_arg.source}|\"."
           end
+
+          [:rebuild, :version_scheme].each do |m|
```

Comment:
Not clear what `m` is here (or below); can you use a better name? `method_name` maybe?